core/data/src/main/java/kr/co/cotton/core/data/detailnews/datasource/RemoteDetailNewsDataSource.kt

```diff
@@ -0,0 +1,14 @@
+package kr.co.cotton.core.data.detailnews.datasource
+
+import org.jsoup.Jsoup
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between